java-shared-config/pom.xml

```diff
@@ -0,0 +1,793 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-shared-config</artifactId>
+  <packaging>pom</packaging>
+  <version>1.6.2-SNAPSHOT</version><!-- {x-version-update:google-cloud-shared-config:current} -->
+  <name>Google Cloud Shared Config</name>
+  <url>https://github.com/googleapis/java-shared-config</url>
+  <description>
+    Shared build configuration for Google Cloud Java libraries.
+  </description>
+  <developers>
+    <developer>
+      <id>chingor13</id>
+      <name>Jeff Ching</name>
+      <email>chingor@google.com</email>
+      <organization>Google</organization>
+      <roles>
+        <role>Developer</role>
+      </roles>
+    </developer>
+  </developers>
+  <organization>
+    <name>Google LLC</name>
+  </organization>
+  <scm>
+    <connection>scm:git:git@github.com:googleapis/java-shared-config.git</connection>
+    <developerConnection>scm:git:git@github.com:googleapis/java-shared-config.git
+    </developerConnection>
+    <url>https://github.com/googleapis/java-shared-config</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <issueManagement>
+    <url>https://github.com/googleapis/java-shared-config/issues</url>
+    <system>GitHub Issues</system>
+  </issueManagement>
+
+  <distributionManagement>
+    <snapshotRepository>
+      <id>sonatype-nexus-snapshots</id>
+      <url>https://google.oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+    <repository>
+      <id>sonatype-nexus-staging</id>
+      <url>https://google.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+  </distributionManagement>
+
+  <licenses>
+    <license>
+      <name>Apache-2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
```

Comment:
I think you need to move this section to the new root parent `native-image-shared-config`, and it doesn't need to be duplicated here as it will be inherited.

Reply:
Good point, done. Moved this content from java-shared-config/pom.xml to native-image-shared-config/pom.xml and removed the duplicated content.